.travis.yml

```diff
@@ -1,22 +1,16 @@
 language: node_js
 node_js:
-  - "4.0.0"
   - "6.0.0"
-  - "4"
   - "6"
+  - "8"
```

Comment:
Could we keep the base versions here too, so add `8.0.0`

Reply:
Sure, I can add that

Reply:
Should we use `8.9.1` as that is the LTS version?

Reply:
Using `8` specifically targets the latest release at build time, which is fine. I quite like the idea of testing against the base version as well